src/System.Private.CoreLib/src/System/Diagnostics/Eventing/RuntimeEventSourceHelper.Linux.cs

```diff
@@ -0,0 +1,72 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Buffers;
+using System.IO;
+
+
+namespace System.Diagnostics.Tracing
+{
+	internal sealed class RuntimeEventSourceHelper
+	{
+		internal static long GetProcessTimes()
+		{
+            long pid = Environment.GetCurrentProcessId();
+            using (FileStream fs = new FileStream($"/proc/{pid}/stat", FileMode.Open, FileAccess.Read, FileShare.Read, bufferSize: 1))
```

Comment:
We have helpers that do all this in corefx, e.g.
https://github.com/dotnet/corefx/blob/b8bc4ff80c5f7baa681e8a569d367356957ba78a/src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs#L195
Should we move that to the shared corelib folder and reuse those routines?

Reply:
None of this should be needed once the rest of the feedback is addresses.

Reply:
> None of this should be needed once the rest of the feedback is addresses.

Ah, great.  I hadn't read the other feedback yet.  I'll just stop commenting on this until it's addressed :)